deploy/charts/rook-ceph/values.yaml

```diff
@@ -54,6 +54,9 @@ pspEnable: true
 # Set the priority class for the rook operator deployment if desired
 # priorityClassName: class
 
+# If true, loop devices are allowed to be used for osd
+allowLoopDevices: false
```

Comment:
This setting needs to be documented in `Documentation/Helm-Charts/operator-chart.md`

Reply:
I updated the document.